lib/game/getGameSorts.js

```diff
@@ -0,0 +1,38 @@
+// Includes
+const http = require('../util/http.js').func
+
+// Args
+exports.required = ['gameSortsContext']
+exports.optional = []
+
+// Docs
+/**
+ * ✅ Get the sorts for a game.
+ * @category Game
+ * @alias getGameSorts
+ * @param {string|number} gameSortsContext - The sort context to get.
```

Comment:
Would prefer defined enums here; I know its tedious given the modern documentation but they are

"GamesDefaultSorts",
"GamesAllSorts",
"HomeSorts",
"ChatSorts"

hilariously, the "updated" docs, list 0 - 7 but only 0 - 3 work without throwing an HTTP 500; thanks Roblox